pkg/webhook/manifests.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	admissionReviewVersions = []string{"v1"}
+	admissionReviewVersions = []string{"v1", "v1beta1"}
```

Comment:
Is this needed?  It looks accidental...  maybe split into another (non-test) PR if it is needed, or add comments etc

Reply:
Not by accident. The mock test hits the webhook that only allows v1 version. My gut feeling is we may need to further separate some of the mock process from kcc code.  I'll dig deeper next week.

Reply:
Fixed. It really took me a while to figure out what happened. In short, [KCC development setup](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/environment-setup/repo-setup.sh) installs an older kube-apiserver (via kubebuilder v2.3.1) and failed to install a new one (in a later step, the e2e test uses) due to my cloudtop permissions.  The old kube-apiserver only supports v1beta1 but not v1 that breaks the webhook admission review.  What confused me and sent me to the wrong direction is that I'm in a new test env (no cluster, no docker, no gcloud default setup) so I was assuming the dependencies were all up to date.